### DISCUSSION TITLE
Добавить маркер, что фильтры включены

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#539] [Есть идея, а что если...]
Иногда стоят фильтры, и это незаметно. Может менять цвет кнопке "очистить фильтры", чтобы было понятно, что какой-то фильтр включен? Спасибо.

![Screenshot_2](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/05de4f95-68bd-4806-8c2d-161ef11934f5)

**Comment by ttm-alexeypolunin** [comment]
Там история такая, что они же могут рассчитываться кодом. Непонятно, что считать включенностью.

Можно конечно предположить так: если нажатие на Ластик приведет к изменению значений в префильтрах, то показывать индикацию. Но для этого надо делать дополнительный расчет строки фильтров, а там может быть кода на 2 секунды и считать надо будет при каждом действии на странице.

Можно сделать расчет подключаемой, но тогда если в одних таблицах будет, а в других не будет — то это тоже так-себе идея тк это супер-стандартное место и пользователей запутает.

**Reply by ttm-alexeypolunin** [reply]
Кстати — если вам надо по каким-то условиям подсветить поле в префильтре, то там работает код форматирования > делаете заливку у поля по условиям, пользователь будет видеть, что там требуется его действие.

**Reply by mihail-cornev** [reply]
Все гениальное - просто, как и сам тотум. Спасибо вам за помощь.
